packages/components/spacings/spacings-inset/src/inset.tsx

```diff
@@ -24,22 +24,24 @@ const getPadding = (scale?: TScale) => {
 
 export type TInsetProps = {
   scale: TScale;
+  height: 'min-content' | 'max-content';
   children?: ReactNode;
 };
 
 const Inset = (props: TInsetProps) => (
   <div
     css={css`
       padding: ${getPadding(props.scale)};
-      height: 100%;
+      height: ${props.height === 'max-content' ? '100%' : 'auto'};
```

Comment:
Ah I thought we will be using the actual value `max-content` or `min-content` instead of `100%` or `auto` 😅

Maybe we can use `100%` and `auto` as the acceptable prop values?

Reply:
Updated here: 374ddc4eaf315cc80fc71b34e06eee4d560f1035

Please let me know if this is enough for you or if there's anything else you want me to update.